### PR TITLE
Normalize event names

### PR DESCRIPTION
**Pull request by edublancas**
closes https://github.com/ploomber/core/issues/38 as discussed in https://github.com/ploomber/jupysql/pull/95#discussion_r1090260969

also added config for building docs from PRs

### CODE REVIEW

File: .github/workflows/rtd.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# .github/workflows/documentation-links.yaml
+
+name: Read the Docs Pull Request Preview
```
Comment (yafimvo) [inline .github/workflows/rtd.yaml:3]: I guess it's not related to the issue

File: doc/telemetry.md [hunk 2]
```diff
@@ -50,6 +50,10 @@ def add(x, y):
 add(1, 41)
 ```
 
+```{note}
+Event names are normalized by replacing underscores (`_`) with hyphens (`-`).
```
Comment (yafimvo) [inline doc/telemetry.md:54]: I think we can clarify and add some examples of how the event name should be formatted.
Later in the tests, we have `somepackage-do_stuff-success` and `somepackage-SomeObject-do_stuff-started` but I'm not sure we actually follow these rules in our codebase. Maybe add it to contributing repo?

**Review by yafimvo — COMMENTED** [review]
lgtm

added a minor note that we can discuss it

File: doc/telemetry.md (hunk 2 above)
Comment (edublancas) [reply doc/telemetry.md:54]: I think we want people not to add custom names, but just add the `log_call` decorator, only in exceptional cases, they should add a custom name. this way we'll control the formatting

**Review by idomic — APPROVED** [review]
looks good

**Pull request merged**